### DISCUSSION TITLE
Flashing 2.3.3 ESC

### DISCUSSION THREAD

**Discussion by PaulinTask** [timeylies/How-to-convert-a-Ninebot-SNSC-2.2-or-2.3-Scooter#5] [Q&A] [answered]
Hi! I have a SNSC 2.3.3 model that has an ESC slightly different than the one shown in your tutorial. Indeed the connection pins are not exactly in the same place but more on the edge of the controller and there is an additional pin that serves for aliementation in 3,3V or 5V. I managed to connect to these pins with the stlink, and the esc starts flashing red/green (I don’t know if this is good?). I guess that I am therefore connect to the esc but I can not connect on the software STM32 ST Link Utility which tells me "Can not connect to target!"
![Capture d’écran 2024-10-08 185450](https://github.com/user-attachments/assets/576ef110-9e5e-4499-8b75-3af3bedcbfb4)

**Comment by timeylies** [comment, marked as answer]
The reason why it doesn’t flash, is because it doesn’t have a stm32 mcu, it most likely has an Artery or Nation mcu, which are clones of the stm32. Try using ScooterHacking Reflasher with the AT32 option enabled. And also, don’t supply 5v, supply 3.3v. And esc should only have the red light on when connected to the esc.
